### DISCUSSION TITLE
Selected GSoC 2022 projects

### DISCUSSION THREAD

**Discussion by ssheorey** [isl-org/Open3D#5113] [GSoC]
We are happy to announce that the following contributors have been chosen under the GSoC 2022 program:

Project Title | Contributor | Assigned Mentor(s)
-- | -- | --
Open3D visualization on cloud notebooks | fanxiaolin | Rene
Open3D-ML 3D Object detection model (PV-RCNN++) | Narayanan E.R. | Sanskar Agrawal
Geometry processing algorithms for tensor-based geometry | Zhengyu | Yixing

Congratulations and welcome abroad! We will be reaching out to you shortly with next steps. 

We were truly happy with the enthusiastic response that Open3D received. If you applied and were not selected, know that it was a tough choice due to the limited number of mentors available, and we wanted to make sure that each selected contributor gets sufficient attention from the Open3D team. We hope you all continue using and following Open3D and apply again next year. We will be happy to continue the conversation about any issues and PRs that you have to contribute to Open3D.

Regards,
Sameer & Yixing

**Comment by kqdidit119** [comment]
Is this Organization participating in GSoC 2023? Will it be there for projects?
